indra/llcommon/lua_function.cpp

```diff
@@ -831,7 +848,8 @@ LuaState::~LuaState()
             // stack contains atexit, debug.traceback()
         }
         // pop debug.traceback()
-        lua_pop(mState, 1);
+        if(debug_traceback)
+            lua_pop(mState, 1);
```

Comment:
And this can test for nonzero `debug.traceback()` stack index, and call `lua_remove(mState, traceback)`.